net/dns/manager_test.go

```diff
@@ -614,7 +755,7 @@ func TestManager(t *testing.T) {
 				SplitDNS:   test.split,
 				BaseConfig: test.bs,
 			}
-			m := NewManager(t.Logf, &f, nil, tsdial.NewDialer(netmon.NewStatic()), nil, nil)
+			m := NewManager(t.Logf, &f, nil, tsdial.NewDialer(netmon.NewStatic()), nil, nil, test.goos)
```

Comment:
i think you need to explicitly set this to `linux` on empty to work with `// empty means "linux"`

Reply:
That's handled in https://github.com/tailscale/tailscale/pull/12334/files#diff-6c214b12264ee689c054e9e0c4c82979d2677d757b79b24e6a7ac45fde4bbe2eR67

Reply:
done